python/paddle/fluid/tests/unittests/test_lrn_op.py

```diff
@@ -154,5 +155,207 @@ def test_errors(self):
+            in_np2 = np.transpose(in_np1, (0, 2, 3, 1))
+
+            in1 = paddle.to_tensor(in_np1)
+            in2 = paddle.to_tensor(in_np2)
+
+            res1 = paddle.nn.functional.local_response_norm(
+                x=in1, size=5, data_format='NCHW')
+            res2 = paddle.nn.functional.local_response_norm(
+                x=in2, size=5, data_format='NHWC')
+
+            res2_tran = np.transpose(res2.numpy(), (0, 3, 1, 2))
+            self.assertTrue(np.allclose(res1.numpy(), res2_tran))
+
+    def check_dygraph_5d_input(self, place):
+        with fluid.dygraph.guard(place):
+            in_np1 = np.random.random([3, 3, 3, 40, 40]).astype("float32")
+            in_np2 = np.transpose(in_np1, (0, 2, 3, 4, 1))
+
+            in1 = paddle.to_tensor(in_np1)
+            in2 = paddle.to_tensor(in_np2)
+
+            res1 = paddle.nn.functional.local_response_norm(
+                x=in1, size=5, data_format='NCDHW')
+            res2 = paddle.nn.functional.local_response_norm(
+                x=in2, size=5, data_format='NDHWC')
+
+            res2_tran = np.transpose(res2.numpy(), (0, 4, 1, 2, 3))
+            self.assertTrue(np.allclose(res1.numpy(), res2_tran))
+
+    def test_dygraph(self):
+        for place in self.places:
+            self.check_dygraph_3d_input(place)
+            self.check_dygraph_4d_input(place)
+            self.check_dygraph_5d_input(place)
+
+
+class TestLocalResponseNormFAPIError(unittest.TestCase):
+    def test_errors(self):
+        with program_guard(Program(), Program()):
+
+            def test_Variable():
+                # the input of lrn must be Variable.
+                x1 = fluid.create_lod_tensor(
+                    np.array([-1, 3, 5, 5]), [[1, 1, 1, 1]], fluid.CPUPlace())
+                paddle.nn.functional.local_response_norm(x1, size=5)
+
+            self.assertRaises(TypeError, test_Variable)
+
+            def test_datatype():
+                x = fluid.data(name='x', shape=[3, 4, 5, 6], dtype="int32")
+                paddle.nn.functional.local_response_norm(x, size=5)
+
+            self.assertRaises(TypeError, test_datatype)
+
+            def test_dataformat():
+                x = fluid.data(name='x', shape=[3, 4, 5, 6], dtype="float32")
+                paddle.nn.functional.local_response_norm(
+                    x, size=5, data_format="NCTHW")
+
+            self.assertRaises(ValueError, test_dataformat)
+
+            def test_dim():
+                x = fluid.data(name='x', shape=[3, 4], dtype="float32")
+                paddle.nn.functional.local_response_norm(x, size=5)
+
+            self.assertRaises(ValueError, test_dim)
+
+
+class TestLocalResponseNormCAPI():
+    def setUp(self):
+        np.random.seed(123)
+        self.places = [fluid.CPUPlace()]
+        if core.is_compiled_with_cuda():
+            self.places.append(fluid.CUDAPlace(0))
+
+    def test_dygraph(self):
+        for place in self.places:
```

Comment:
why don't use the new api to test dygraph?

Reply:
Done